src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int figuresNumber = 20;
```

Comment:
I left a comment that this should be moved into a constant field - this was not fixed